.github/workflows/ssh-integrationtest.yml

```diff
@@ -0,0 +1,22 @@
+# Run the ssh integration tests with `make sshintegrationtest`.
```

Comment:
While I'm in here, I'm adding the integration tests that I've run manually to CI. They'll only run if someone touches the SSH code.